programs/zstdcli.c

```diff
@@ -1244,10 +1244,6 @@ int main(int const argCount, const char* argv[])
         CLEAN_RETURN(1);
     }
 
-    /* No status message in pipe mode (stdin - stdout) or multi-files mode */
-    if (!strcmp(filenames->fileNames[0], stdinmark) && outFileName && !strcmp(outFileName,stdoutmark) && (g_displayLevel==2)) g_displayLevel=1;
```

Comment:
I really believe that reducing verbosity to quiet `-q` mode when `zstd` is used in a pipe configuration
remain the correct thing to do.

The goal of this patch is to add a status line when there are multiple input files.
So only the second line (below) should be removed.